lumen/transforms/sql.py

```diff
@@ -188,7 +188,8 @@ def apply(self, sql_in):
             elif isinstance(val, list):
                 if not val:
                     continue
-                condition = f'{col} IN {tuple(val)!r}'
+                val = [f'\'{v}\'' for v in val]
+                condition = f'''{col} IN ({', '.join(val)})'''
```

Comment:
Can you explain this change to me? The resultant string is identical in all cases I can identify.

Reply:
Nevermind, I see the issue. This also does not seem quite correct though.